docs/sources/access/index.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Access or install Explore Profiles.
+_build:
+  list: false
+noindex: true
```

Comment:
We need instructions for how to install or access Explore Profiles. 

This page doesn't appear in the build. It's here as a placeholder until we write the content.

Reply:
Done.